src/suites/positive/validation/positive-validator.ts

```diff
@@ -1,28 +1,29 @@
 import { isEqual, uniqWith } from 'lodash';
 import { Json, SchemaObject } from 'swagger-client/schema';
-import ValidationMessage, { Type } from '../validation-message';
+import { BaseValidator } from '../../../validation';
+import { Message } from '../../../validation';
+import PositiveMessage, { Type } from './positive-message';
 
-abstract class BaseValidator {
-  protected _failures: Map<string, ValidationMessage>;
-  protected _warnings: Map<string, ValidationMessage>;
-  protected validated: boolean;
+abstract class PositiveValidator extends BaseValidator {
+  protected _failures: Map<string, Message>;
+  protected _warnings: Map<string, Message>;
 
   constructor() {
-    this.validated = false;
+    super();
     this._failures = new Map();
     this._warnings = new Map();
   }
 
-  public get failures(): Map<string, ValidationMessage> {
+  public get failures(): Map<string, Message> {
     return this._failures;
   }
 
-  public get warnings(): Map<string, ValidationMessage> {
+  public get warnings(): Map<string, Message> {
     return this._warnings;
   }
 
-  protected addMessage(type: Type, path: string[], props?: string[]): void {
-    const message = new ValidationMessage(type, path, props);
+  public addMessage(type: unknown, path: string[], props?: string[]): void {
```

Comment:
This is a ongoing issue I've observed before with the current approach used for Jest testing since many methods are assigned as public then they should be private/protected in order to better support unit testing.

In this case it was also done to match the other validator.